check-new-tool-versions.ps1

```diff
@@ -0,0 +1,127 @@
+<#
+.SYNOPSIS
+Check and return list of new available tool versions
+#>
+
+param (
+    [Parameter(Mandatory)] [string] $DistURL,
+    [Parameter(Mandatory)] [string] $ManifestLink,
+    [string] $VersionFilterToInclude,
```

Comment:
`[string[]]`
and you won't need to do `$versionFilters = $VersionFilter.Split(',')`